src/montecarlo/run_fund_scc_mcs.jl

```diff
@@ -0,0 +1,97 @@
+using Mimi
+
+include(joinpath(@__DIR__, "defmcs.jl"))                  # Defines FUND's distributional parameters
+include(joinpath(@__DIR__, "../new_marginaldamages.jl"))  # Defines FUND's marginal emissions model
+
+"""
+Runs a Monte Carlo simulation with the FUND model over it's distirbutional parameters, and calculates
+the Social Cost of Carbon for the specified `years` and discount `rates`.
+`trials`: the number of trials to run.
+`ntimesteps`: how many timesteps to run.
+`output_dir`: an output directory; if none provided, will create and use "output/SCC yyyy-mm-dd HH-MM-SS MCtrials". 
+`save_trials`: whether or not to generate and save the MC trial values up front to a file.
+"""
+function run_fund_scc_mcs(trials = 10000; years = [2020], rates = [0.03], ntimesteps = 1051, output_dir = nothing, save_trials = false)
+
```

Comment:
1051 must be the size of something else. Shouldn't you compute this from that, rather than assuming this size?